components/taxonomy/src/values.rs

```diff
@@ -1556,19 +927,26 @@ impl TimeStamp {
         TimeStamp(date)
     }
 }
-impl Parser<TimeStamp> for TimeStamp {
+
+impl Data for TimeStamp {
     fn description() -> String {
-        "TimeStamp".to_owned()
+        "TimeStamp (RFC 3339)".to_owned()
     }
-    fn parse(path: Path, source: &JSON) -> Result<Self, ParseError> {
+    fn parse(path: Path, source: &JSON, _binary: &BinarySource) -> Result<Self, Error> {
```

Comment:
wondering why _binary is here at all, I don't see where it's used ? but maybe I missed something.